nautobot/core/ui/object_detail.py

```diff
@@ -699,6 +699,7 @@ def __init__(
         footer_content_template_path="components/panel/footer_content_table.html",
         footer_buttons=None,
         form_id=None,
+        include_paginator=False,
```

Comment:
This is a change to the exposed API, it makes sense, but just calling out for core to review if they agree.